tokio-postgres/src/connect.rs

```diff
@@ -408,7 +418,7 @@ async fn check_and_refresh(config: &Config) -> bool {
                     &mut NoTls,
                     config,
                 )
-                .await
+                    .await
```

Comment:
Proper formatting change.

Reply:
Some of these formatting changes are part of the code which is coming from upstream fork. We try to not make many changes in the upstream code to keep conflicts to a minimum for the next planned upgrade.

Please revert all these for this PR, we will take care of these in our next version upgrade.
Thanks

Reply:
Reverted.